### ISSUE TITLE
composite characters (hindi)

### ISSUE THREAD

**Issue by vedadeepta** [quilljs/quill#3425]
**Steps for Reproduction**

1. Visit https://quilljs.com/
2. Remove everything in the example RTE editor on the main page
3. Paste `फ़` and then press backspace 

**Expected behavior**:

Removes the whole character

**Actual behavior**:

Character `़` is shown after hitting backspace

**Platforms**:
Chrome `92.0.4515.131 (Official Build) (x86_64)`
OSX

**Version**:

Run `Quill.version` to find out
`1.3.7`

Tried the same steps with version `2.x-dev` and it works as expected.

**Comment by vedadeepta** [comment]
yes i think so, the character `फ़` is represented by a single unicode `\u095e` but the editor interprets it as the composite of two unicodes (`\u092b\u093c`) instead of a single unicode.

**Comment by benbro** [comment]
Please test [v2.0.0-beta.1](https://github.com/quilljs/quill/releases/tag/v2.0.0-beta.1)
https://v2.quilljs.com/standalone/full
